test/conformance/kernel/urKernelSetExecInfo.cpp

```diff
@@ -56,7 +56,7 @@ struct urKernelSetExecInfoUSMPointersTest : uur::urKernelTest {
 UUR_INSTANTIATE_KERNEL_TEST_SUITE_P(urKernelSetExecInfoUSMPointersTest);
 
 TEST_P(urKernelSetExecInfoUSMPointersTest, SuccessHost) {
-    bool host_supported = false;
+    ur_device_usm_access_capability_flags_t host_supported = false;
```

Comment:
`= false` might compile but it should really be `0`. same goes for all the other instances of this check in this file.